React/TeamMemberCard/src/TeamMemberCard.jsx

```diff
@@ -0,0 +1,20 @@
+export function TeamMembers(props) {
+    return(
+        <div        
+            style={{
```

Comment:
This is genenerally a bad practice and thus should be avoided. `style` attribute has a higher specificity then classes and it's not reusable. Please create a class and then use the class name here.


```suggestion
            class="team-member-container"
```

Then in the css file,

```suggestion
            .team-member-container {
               max-width: 200px;
               // other styles
            }
```